mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -946,14 +966,31 @@ static FuncOp createVerifyFuncOp(ModuleOp &module, OpBuilder &builder,
                                     cpuLoadOp, gpuLoadOp);
     ifOp = bt4.create<scf::IfOp>(builder.getUnknownLoc(), cmpOp, false);
   } else {
-    auto cmpOp = bt4.create<CmpFOp>(builder.getUnknownLoc(), CmpFPredicate::UNE,
-                                    cpuLoadOp, gpuLoadOp);
-    ifOp = bt4.create<scf::IfOp>(builder.getUnknownLoc(), cmpOp, false);
+    if (randomSeed.getValue() != "none" &&
+        randomDataType.getValue() == "float") {
+      float delta = 0.0000001;
+      if (outputMemRefType.getElementType() == builder.getF16Type())
+        delta = 0.0001;
```

Comment:
in the future PRs we might need `delta` be specified, but let's leave it for now.